MODULE.bazel

```diff
@@ -595,13 +592,36 @@ dev_maven.artifact(
 
 # https://github.com/bazelbuild/rules_jvm_external/issues/865
 dev_maven.artifact(
-    name = "regression_testing",
+    name = "regression_testing_coursier",
     artifact = "google-api-services-compute",
     classifier = "javadoc",
     group = "com.google.apis",
     version = "v1-rev235-1.25.0",
 )
 
+# These artifacts helped discover limitations in the Maven resolver. Each
+# artifact listed here *must have* an accompanying issue. We build_test these
+# targets to ensure that they remain supported by the rule.
+dev_maven.install(
+    name = "regression_testing_maven",
+    artifacts = [
+        # Depends on org.apache.yetus:audience-annotations:0.11.0 which has an invalid pom
+        "org.apache.parquet:parquet-common:1.11.1",
+        # To match Maven's behavior the resolver should not error when there is an artifact
```

Comment:
For top-level artifacts, this seems like something we'd want to fail on. For transitive deps, the warning seems sensible.

Reply:
Yeah, that makes sense.  I'll try updating it so it throws the exception if the missing artifact is in dependency list.